src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -58,7 +54,7 @@ Once the Slack app is configured and you are authenticated, select the Slack cha
 
 ## Built-in screenshot sharing [#built-in-screenshots]
 
-With our new integrated screenshot selection tool you can attach screenshots to your comments. Just click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 
+Use our integrated screenshot selection tool you can attach screenshots to your comments. Just click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 
```

Comment:
```suggestion
Attach screenshots to your comments by clicking our integrated screenshot selection tool. Just click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 
```

clicking or using for the verb. But makes this more action-specific and frontloads what outcome the user may want!